src/modules/Home.controller.js

```diff
@@ -34,14 +48,14 @@ export const renderMovieHandler = (Database) => currentData(Database).map((movie
         <div class="container">
         <div class="card-header">
             <Label>${movie.name}</Label>
-            <span class="material-symbols-outlined">
+            <span  id=${movie.id} class="material-symbols-outlined ">
             favorite 
             </span>
             </div>
         <ul>
             <li>${movie.premiered}</li>
             <li>${movie.averageRuntime} min</li>
-            <li>0 Likes</li>
+            <li>${movie.likes ? movie.likes : '0'} Likes</li>
             </ul>
             <button id=${movie.id}>Comment</button>
             <button id=${movie.id}>Reservation</button>
```

Comment:
You may avoid the comment and reservation buttons having the same ID as this is not ideal.

Attaching a prefix could be an option. e.g `comment_${movie.id}` or `reserve_${movie.id}` . To get back the ID, one can split the string by '_' and select the item at index 1.